Core/GDCore/IDE/PropertyFunctionGenerator.cpp

```diff
@@ -122,13 +130,25 @@ void PropertyFunctionGenerator::GenerateGetterAndSetter(
           .SetDescription("Change " + descriptionSubject)
           .SetSentence("_PARAM0_ " + UnCapitalizeFirstLetter(propertyLabel) +
                        ": _PARAM2_");
-      gd::ParameterMetadata parameter;
-      parameter.SetType("yesorno")
+      gd::ParameterMetadata objectParameter;
+      objectParameter.SetType("object")
+          .SetName("Object")
+          .SetDescription("Object")
+          .SetExtraInfo(eventsBasedBehavior.GetObjectType());
+      gd::ParameterMetadata behaviorParameter;
+      behaviorParameter.SetType("behavior")
+          .SetName("Behavior")
+          .SetDescription("Behavior")
+          .SetExtraInfo(behaviorFullType);
+      gd::ParameterMetadata valueParameter;
+      valueParameter.SetType("yesorno")
           .SetName("Value")
           .SetDescription(capitalizedName)
           .SetOptional(true)
           .SetDefaultValue("yes");
-      setter.GetParameters().push_back(parameter);
+      setter.GetParameters().push_back(objectParameter);
+      setter.GetParameters().push_back(behaviorParameter);
+      setter.GetParameters().push_back(valueParameter);
```

Comment:
This is the real change.
Other changes are for a refactor to factorize some helper methods to build full type strings.